src/Microsoft.AspNet.Authentication.OpenIdConnect/DefaultIdentityModelLoggingListener.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Diagnostics.Tracing;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Authentication.OpenIdConnect
+{
+    /// <summary>
+    /// Default logging event listener for wilson event source. It logs the data to the target where other Asp.Net logs go.
+    /// </summary>
+    public class DefaultIdentityModelLoggingListener : EventListener
```

Comment:
I am not sure if I understand this. Wilson uses IdentityModelEventSource to log things. To make it log into the same place as aspnet, I need to hook an eventListener to override the default behavior in Wilson and tell it to redirect logs to aspnet logger. How else can I integrate the IdentityModelEventSource in aspnet? Or are you suggesting not using the IdentityModelEventSource at all?